include/drivers/ti/uart/uart_16550.h

```diff
@@ -23,6 +23,8 @@
 #define UARTMSR			0x18
 #define UARTSPR			0x1c
 #define UARTCSR			0x20
+/* Some instances have MDR1 defined as well */
+#define UARTMDR1		0x20
```

Comment:
minor: It looks like all these register definitions after 0x1c are not defined in the spec you refer to, nor are they used in the code. Perhaps they should be removed (apart from the UARTMDR1 you added)?

Reply:
They are available in some implementations, they most likely will not be used like most of these registers, but for completeness I see no reason to drop them here.